README.md

```diff
@@ -50,3 +50,18 @@ sudo pip install -r requirements.txt
 sudo pip install -r dev_requirements.txt
 cp greenpithumb/wiring_config.ini.example greenpithumb/wiring_config.ini
 ```
+
+## QuickStart Instructions
+
+Run greenpithumb/greenpithumb.py and supply command-line flags for poll_interval, photo_interval, config_file, sleep_window, image_path, and db_file. Example below.
+
+```bash
+python greenpithumb/greenpithumb.py \
+  --poll_interval 15 \
+  --photo_interval 240 \
+  --config_file "C:\GreenPiThumb\greenpithumb\config.ini" \
```

Comment:
If they followed the install directions, they don't actually have to specify this and it's probably not an interesting parameter for most people. Suggest dropping from the example.